modules/offers/client/views/directives/tr-offer-host-view.client.view.html

```diff
@@ -198,6 +198,11 @@ <h4 ng-if="!trOfferHost.isOwnOffer">
            class="btn btn-sm btn-inverse-primary">
           Bigger map
         </a>
+        <a ng-if="trOfferHost.isMobile"
+          ng-href="geo:{{trOfferHost.offer.location[0]}},{{trOfferHost.offer.location[1]}};u=200"
+           class="btn btn-sm btn-inverse-primary">
+          Open on device
```

Comment:
Feels very technical; would there be any alternative ways to say the same?

Not a blocker though, we can very well merge with this. :-)

Reply:
`See location in app` maybe? Although I think I prefer `Open on device`

Reply:
We can think about it and update later on. 👍